packages/patternfly-react/src/components/VerticalNav/index.js

```diff
@@ -1,29 +1,23 @@
 import VerticalNav from './VerticalNav';
-import VerticalNavMasthead from './VerticalNavMasthead';
 import VerticalNavItem from './VerticalNavItem';
 import VerticalNavSecondaryItem from './VerticalNavSecondaryItem';
 import VerticalNavTertiaryItem from './VerticalNavTertiaryItem';
-import VerticalNavBrand from './VerticalNavBrand';
 import VerticalNavIconBar from './VerticalNavIconBar';
 import VerticalNavBadge from './VerticalNavBadge';
 import VerticalNavDividerItem from './VerticalNavDividerItem';
 
-VerticalNav.Masthead = VerticalNavMasthead;
 VerticalNav.Item = VerticalNavItem;
 VerticalNav.SecondaryItem = VerticalNavSecondaryItem;
 VerticalNav.TertiaryItem = VerticalNavTertiaryItem;
-VerticalNav.Brand = VerticalNavBrand;
 VerticalNav.IconBar = VerticalNavIconBar;
 VerticalNav.Badge = VerticalNavBadge;
 VerticalNav.Divider = VerticalNavDividerItem;
 
 export {
   VerticalNav,
-  VerticalNavMasthead,
```

Comment:
This is a breaking change.  Need to update the commit to have the breaking changes flag so that semantic will do a major. @jeff-phillips-18 Since we will need a major is there anything else we want to try to get in?  I know bindMethods needs removed so we can get rid of the deprecation message.

Reply:
Can we adjust to make this non-breaking?